azurerm/internal/services/frontdoor/resource_arm_frontdoor.go

```diff
@@ -67,6 +67,13 @@ func resourceArmFrontDoor() *schema.Resource {
 				Required: true,
 			},
 
+			"backend_pools_send_receive_timeout_seconds": {
```

Comment:
Should this be `backend_pool_send_receive_timeout_in_seconds`?

Reply:
I was originally going to call it that too, but after researching it, it's a global setting for [all backend pools](https://docs.microsoft.com/en-us/rest/api/frontdoorservice/frontdoor/frontdoors/get#backendpoolssettings) thus the plural. 🙂

Reply:
ahh see i would have assumed it was applying to all of them without the s, but there is another property backend_pool? which is different the the other back end pools i can see why heh